src/EditorFeatures/CSharpTest/ChangeSignature/AddParameterTests.cs

```diff
@@ -377,6 +377,34 @@ public MyAttribute(int y, byte b, string x)
             await TestChangeSignatureViaCommandAsync(LanguageNames.CSharp, markup, updatedSignature: permutation, expectedUpdatedInvocationDocumentCode: updatedCode);
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.ChangeSignature)]
+        public async Task AddOptionalParameterWithOmittedCallsiteToAttributeConstructor()
```

Comment:
nit: why is this not in the Omitted file?

Reply:
Good nit. Fixed.